Let's help to build MIMIC-IV-ED PostgreSQL locally

JackieMium opened the pull request:
I am not really an expert on PostgreSQL, this is what I can do currently to try load mimic-iv-ed 1.0 data and build postgresql databse locally. Scripts were modified from `mimic-iii/buildmimic/postgres/`
Still need to add following files:

- `postgres_add_comments.sql`
- `postgres_add_constraints.sql`
- `postgres_add_indexes.sql`
- `postgres_checks.sql`

but I am just opening a PR this early to welcome any help with it.

briangow commented:
@JackieMium thanks so much for taking the initiative to develop MIMIC-IV-ED local build scripts! I'm sure the community will appreciate having them.  

In thinking about this, I was wondering if it makes more sense to take the MIMIC-III approach of using a Makefile or instead use the [MIMIC-IV approach](https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/buildmimic/postgres) for consistency and simplicity.  What is your thought?

JackieMium commented:
> In thinking about this, I was wondering if it makes more sense to take the MIMIC-III approach of using a Makefile or instead use the [MIMIC-IV approach](https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/buildmimic/postgres) for consistency and simplicity. What is your thought?

I'd vote for the MIMIC-IV approach as it's platform independent, more intuitive.

briangow commented:
Agreed. I'm assuming you'll update your pull request to align with the MIMIC-IV approach when you have the time. Please let us know if you won't be able to do that though. Thanks again for your contributions!

JackieMium commented:
@briangow I've changed the scripts and renamed them accordingly, README was also modified. We still need `index.sql` and `constraint.sql`, but I am not that experienced with Postgres so I am leaving this pull reqeust here to call for help with it.

alistairewj commented:
I updated this with some changes based on what I know. My hunch is it should work.. I did not add indices as I don't think they are useful for data this small.

@briangow maybe you can run through the README and see how well that acts as guidance? I didn't really look through it.
